drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1,69 +1,114 @@
-function DragAndDropBlock(runtime, element) {
-
+function DragAndDropBlock(runtime, element, configuration) {
+    "use strict";
+    // Ensure "undefined" has not been redefined (though this unlikely and often impossible).
```

Comment:
@mtyaka I knew that, and was about to reply that it's mostly `blah.blah` in this file, but I just realized that I made that mistake on line 7 below :-/
Thanks for the reminder :)